src/test/resources/json/_initial.json

```diff
@@ -0,0 +1,15 @@
+{
+  "a": 1,
+  "b": "string",
+  "c": [3, 2, 1],
+  "d": {
```

Comment:
I suggest adding more levels of deeply nested objects (let's say, 15 levels) to ensure flattener doesn't fail with stack overflow exception

Also, add in nested objects duplicated keys to ensure flattener handles this properly

Reply:
@tdorzhi what do you mean by duplicated keys? Like this?

```
{
  "a": 1,
  "a": 2
}
```

Reply:
no, I mean something like this:
```
{ 
  "somekey": {
     "somekey": {
         "somekey": "somevalue"
      }
   }
}
```

Reply:
Done